examples/object_subclass/author.rs

```diff
@@ -23,9 +23,8 @@ mod imp {
     #[glib::derived_properties]
     impl ObjectImpl for Author {
         fn signals() -> &'static [Signal] {
-            static SIGNALS: Lazy<Vec<Signal>> =
```

Comment:
I wonder if we shouldn't wait for stabilization of `LazyLock`. But this also seems simple enough, doesn't really make much difference the way it's used in most places.

@bilelmoussaoui What do you think?

Reply:
`LazyLock` will not make it any cleaner or shorter in such cases.

- You need two statements with `OnceLock`: a declaration and an acquiring with initialization.
- You need two statements with `LazyLock`: a declaration with initialization and an acquiring.

OTOH, Initialization of `Regex`es with `LazyLock` would be much cleaner.

Reply:
I am fine with this change, at the end of the day, the ideal situation would be having a macro for generating the signals boilerplate.